app/services/organization_service.py

```
@@ -0,0 +1,432 @@
+        org_id: str,
+        requesting_user_id: str,
+        target_user_id: str,
+        role: str = "member",
+    ) -> Optional[OrgMembership]:
+        """Add a member to the organization. Requires owner or admin role."""
+        membership = await self._get_membership(org_id, requesting_user_id)
+        if not membership or membership.role not in ("owner", "admin"):
+            return None
+
+        if role not in ("owner", "admin", "member", "viewer"):
+            raise ValueError(f"Invalid role: {role}")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)